### PR TITLE
Bug 1797045: [3.11] Clean up stale egress IP iptables rules on startup

### PR DESCRIPTION
**Pull request by danwinship**
Backport of https://github.com/openshift/sdn/pull/88

### CODE REVIEW

**Comment by danwinship** [comment]
/assign @knobunc @juanluisvaladas 
(Ben reviewed the master PR, openshift/sdn#88, Juan reviewed the 4.3 PR, openshift/sdn#100.)

File: pkg/network/node/networkpolicy_test.go [hunk 1]
```diff
@@ -550,7 +550,7 @@ func _TestNetworkPolicyCache(t *testing.T) {
 		name := fmt.Sprintf("namespace-%d", vnid)
 		addNamespace(np, name, vnid, map[string]string{
 			"pod.network.openshift.io/legacy-netid": fmt.Sprintf("%d", vnid),
-			"name": name,
+			"name":                                  name,
```
Comment (juanluisvaladas) [inline pkg/network/node/networkpolicy_test.go]: Looks like there is a formatting issue here:

File: pkg/network/node/networkpolicy_test.go [hunk 2]
```diff
@@ -638,7 +638,7 @@ func _TestNetworkPolicyCache(t *testing.T) {
...
```
Comment (juanluisvaladas) [inline pkg/network/node/networkpolicy_test.go]: same

File: pkg/network/node/networkpolicy_test.go [hunk 3]
```diff
@@ -647,7 +647,7 @@ func _TestNetworkPolicyCache(t *testing.T) {
...
```
Comment (juanluisvaladas) [inline pkg/network/node/networkpolicy_test.go]: same

**Comment by danwinship** [comment]
> Looks like there are a few unintended changes in networkpolicy_test.go, I assume these were commited accidentally.

ah, no, it seems that they're necessary to make test-verify pass. I'm not sure how the branch ended up in the current state...

anyway, repushed as two separate commits

**Comment by juanluisvaladas** [comment]
You're right :-|
/lgtm
/retest

**Comment by danwinship** [comment]
> ah, no, it seems that they're necessary to make test-verify pass. I'm not sure how the branch ended up in the current state...

ok, I think I was comparing the wrong test results vs pushes or something, because it seems like actually that part of the patch was *breaking* test-verify... anyway, reverted now

**Comment by danwinship** [comment]
@juanluisvaladas this seems to have passed now. can you re-lgtm?

**Pull request merged**